src/tox_uv/_venv.py

```diff
@@ -45,10 +46,20 @@ def register_config(self) -> None:
         super().register_config()
         desc = "add seed packages to the created venv"
         self.conf.add_config(keys=["uv_seed"], of_type=bool, default=False, desc=desc)
+        self.conf.add_config(
```

Comment:
We should not redefine use what tox already defines https://tox.wiki/en/4.16.0/config.html#skip_missing_interpreters

Reply:
Sorry, I'm starting with the tox API :grimacing: 

I tried to access the `skip_missing_interpreters` through `self.conf` but the key was not there. Do you have an idea how to retrieve it ?

Reply:
Found it! Need to use `self.core`